CITATION.cff

```diff
@@ -0,0 +1,77 @@
+# This CITATION.cff file was generated with cffinit.
+# Visit https://bit.ly/cffinit to generate yours today!
+
+cff-version: 1.2.0
+title: >-
+  XCP-D : A Robust Postprocessing Pipeline of fMRI
+  data
+message: >-
+  If you use this software, please cite it using the
+  metadata from this file.
+type: software
+authors:
+  - given-names: Azeez
+    family-names: Adebimpe
+    affiliation: University of Pennsylvania
+    orcid: 'https://orcid.org/0000-0001-9049-0135'
+  - given-names: Maxwell
+    family-names: Bertolero
+    orcid: 'https://orcid.org/0000-0002-2691-3698'
+    affiliation: University of Pennsylvania
+  - given-names: Kahini
+    family-names: Mehta
+    email: kahini.mehta@pennmedicine.upenn.edu
+    affiliation: University of Pennsylvania
+    orcid: 'https://orcid.org/0000-0001-9466-100X'
+  - given-names: Kristin
+    family-names: Murtha
+    affiliation: University of Pennsylvania
+  - given-names: Matthew
+    family-names: Cieslak
+    email: matthew.cieslak@pennmedicine.upenn.edu
+    affiliation: University of Pennsylvania
+    orcid: 'https://orcid.org/0000-0002-1931-4734'
+  - given-names: Steven
+    family-names: Meisler
+    orcid: 'https://orcid.org/0000-0002-8888-1572'
+    affiliation: >-
+      Harvard-MIT Division of Health Sciences and
+      Technology
+  - given-names: Thomas
+    family-names: Madison
+    affiliation: University of Minnesota
+    orcid: 'https://orcid.org/0000-0003-3030-6580'
+  - given-names: Valerie
+    family-names: Sydnor
+    affiliation: University of Pennsylvania
+    orcid: 'https://orcid.org/0000-0002-8640-668X'
+  - given-names: Sydney
+    family-names: Covitz
+    affiliation: University of Pennsylvania
+    orcid: 'https://orcid.org/0000-0002-7430-4125'
+identifiers:
+  - type: doi
+    value: 10.5281/zenodo.5139942
```

Comment:
The one thing I don't have a solution for is updating the Zenodo DOI. The version-specific DOI is created when the GitHub release is published, but we would need to update the Zenodo DOI in the CITATION.cff file before the release if we want folks to cite version-specific DOIs...